### PR TITLE
@ #24 | update to used teracy-dev-essential v0.3.0

### PR DESCRIPTION
**Pull request by hieptranquoc**
connect #24 
please check this PR! Thank you

### CODE REVIEW

File: manifest.yaml [hunk 1]
```diff
@@ -6,4 +6,4 @@ dependencies:
   - name: teracy-dev-core
     require_version: ">= 0.4.0"
   - name: teracy-dev-essential
-    require_version: ">= 0.1.0"
+    require_version: ">= 0.3.0"
```
Comment (hoatle) [inline manifest.yaml:9]: need to update README

**Comment by hieptranquoc** [comment]
@hoatle I've updated! Please check this PR again! Thank you!

File: README.md [hunk 2]
```diff
@@ -51,7 +51,7 @@ teracy-dev:
           remote:
             origin: https://github.com/teracyhq-incubator/teracy-dev-v05-compat.git
           branch: master
-      require_version: ">= 0.2.0"
+      require_version: ">= 0.3.0"
```
Comment (hoatle) [inline README.md]: keep it as is, it will be updated when released

File: README.md [hunk 3]
```diff
@@ -68,7 +68,7 @@ teracy-dev:
           remote:
             origin: https://github.com/teracyhq-incubator/teracy-dev-v05-compat.git
           branch: develop
-      require_version: ">= 0.3.0-SNAPSHOT"
+      require_version: ">= 0.4.0-SNAPSHOT"
```
Comment (hoatle) [inline README.md]: keep it as is, it will be updated when released

File: README.md [hunk 4]
```diff
@@ -94,5 +94,5 @@ teracy-dev:
             origin: git@github.com:hoatle/teracy-dev-v05-compat.git # your forked repo
             upstream: git@github.com:teracyhq-incubator/teracy-dev-v05-compat.git
           branch: develop
-      require_version: ">= 0.3.0-SNAPSHOT"
+      require_version: ">= 0.4.0-SNAPSHOT"
```
Comment (hoatle) [inline README.md]: keep it as is, it will be updated when released

**Review by hoatle — CHANGES_REQUESTED** [review]
@hieptranquoc please check my comments

**Comment by hieptranquoc** [comment]
@hoatle I applied your comment! Please check again! Thank you

**Review by hoatle — APPROVED** [review]
LGTM

**Pull request merged**